robosuite/controllers/config/ee_pos.json

```diff
@@ -0,0 +1,16 @@
+{
+  "type": "EE_POS",
+  "control_range_pos": 0.05,
+  "damping_max": 2,
+  "damping_max_abs_delta": 0.1,
+  "damping_min": 0,
+  "initial_damping": 1,
+  "initial_impedance_ori": 150,
+  "initial_impedance_pos": 150,
+  "kp_max": 300,
+  "kp_max_abs_delta": 10,
+  "kp_min": 10,
+  "impedance_flag": false,
+  "use_delta_impedance": false,
+  "interpolation": "linear"
```

Comment:
is there a reason that the interpolation is linear here but null in other configs?